examples/cpp_sycl/makefile_lnx

```diff
@@ -135,7 +135,7 @@ sointel64:
 _make_ex: $(RES)
 
 vpath
-vpath %.cpp $(addprefix ./source/,datasource covariance linear_regression logistic_regression optimization_solvers pca kmeans moments)
+vpath %.cpp $(addprefix ./source/,datasource data_management covariance linear_regression logistic_regression optimization_solvers pca kmeans moments)
```

Comment:
Probably we should locate an example for `syclSOANumericTable` (in datasource folder) and example on USM in the same folder